services/player-service/application/src/main/java/com/kjeldsen/player/application/usecases/UpdatePlayerPositionTendencyUseCase.java

```diff
@@ -0,0 +1,21 @@
+package com.kjeldsen.player.application.usecases;
+
+import com.kjeldsen.player.application.repositories.PlayerPositionTendencyWriteRepository;
+import com.kjeldsen.player.domain.PlayerPositionTendency;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@AllArgsConstructor
+@Component
+public class UpdatePlayerPositionTendencyUseCase {
+
+    private PlayerPositionTendencyWriteRepository playerPositionTendencyWriteRepository;
+
+    public PlayerPositionTendency update(UpdatePlayerTendencies updatePlayerTendencies) {
+        PlayerPositionTendency playerPositionTendency = PlayerPositionTendency.builder()
```

Comment:
PlayerPositionTendency.builder()
            .position(updatePlayerTendencies.getPosition())
            .tendencies(updatePlayerTendencies.getTendencies())
            .build();
            
            either a mapper of a method within PlayerPositionTendency.from(UpdatePlayerTendencies upda)

Reply:
Mappers will come in the future.